### ISSUE TITLE
Writing Library Tag

### ISSUE THREAD

**Issue by Abhi5033** [quodlibet/mutagen#644]
This maybe not actually an issue

After muxing a flac file using ffmpeg, I'm trying to add metadata using Mutagen. Post processing in mutagen, the file is still showing Writing Library as ffmpeg in the mediainfo instead of Mutagen.

What I want
![image](https://github.com/quodlibet/mutagen/assets/118114270/bac8a350-c682-4b78-977a-2c5d0329444f)

What I'm getting
![image](https://github.com/quodlibet/mutagen/assets/118114270/ac321d30-5a2b-459d-b9f6-6a8472d5aa8c)

**Comment by phw** [comment]
Indeed, currently Mutagen does not update the vendor string on the vorbis comments, it just sets it when there is no Vorbis comment block yet in the file and mutagen creates a new one. See https://github.com/quodlibet/mutagen/blob/main/mutagen/_vorbis.py#L73

You could set it in your code, though. I think the following should work:

```
audio.tags.vendor = "Mutagen " + mutagen.version_string
```

Should mutagen update it by default every time it writes? Probably, I'm not sure. I personally would say yes, but some users might want to preserve the current behavior.

I think it definitely should update it when calling `VComment.clear`.

**Comment by Abhi5033** [comment]
> Indeed, currently Mutagen does not update the vendor string on the vorbis comments, it just sets it when there is no Vorbis comment block yet in the file and mutagen creates a new one. See https://github.com/quodlibet/mutagen/blob/main/mutagen/_vorbis.py#L73
> 
> You could set it in your code, though. I think the following should work:
> 
> ```python
> audio.tags.vendor = "Mutagen " + mutagen.version_string
> ```
> 
> Should mutagen update it by default every time it writes? Probably, I'm not sure. I personally would say yes, but some users might want to preserve the current behavior.
> 
> I think it definitely should update it when calling `VComment.clear`.

Thanks for the reply sir
But how did that guy achieve that writing library properly? [I found that file in some telegram channels]
I even tried re-writing the audio stream from source file to another flac file but still the ffmpeg tag is carried with it.
